mavros_extras/msg/DistanceSensor.msg

```diff
@@ -0,0 +1,14 @@
+# DISTANCE_SENSOR message data
+
+Header header
+
+sensor_msgs/Range range
+
+uint8 type			# see MAV_DISTANCE_SENSOR enum
+uint8 MAV_DISTANCE_SENSOR_LASER = 0
+uint8 MAV_DISTANCE_SENSOR_ULTRASOUND = 1
+
+uint8 id 			# XXX FIXME enum ??
+
+uint8 orientation 	# rad
+uint8 covariance	# 1E-2 meters; 0 for unknown/invalid readings
```

Comment:
https://github.com/mavlink/c_library/blob/master/common/mavlink_msg_distance_sensor.h#L14 I'm just using the definition. But I agree with `variance`, since we are talking about one unique dimension, which is `z`. Unless we consider the derivative and also consider speed measurements, which considers also time and `vz`. So, should I change to `variance` then and warn PX4 dev team to change message definition?